fcrepo-clients/src/test/java/edu/unc/lib/dl/event/RepositoryPremisLoggerIT.java

```diff
@@ -166,4 +169,65 @@ public void getEventsModelForObjectWithoutLog() throws Exception {
 
         assertTrue(eventsModel.isEmpty());
     }
+
+    @Test
+    public void makeMultipleChangesSimultaneously() throws InterruptedException {
```

Comment:
Good work on this test. Do you know if it fails without the new pid locking code? It can sometimes be hard to trigger concurrency cases with a small number of requests. 

It might be good to check the premis log after all the threads have completed to verify that all the expected events are present at once, just to make sure we end up with a merger of the two change sets at the end.

Its too bad we're using a mock binary transfer session in all these tests, so we aren't getting a test of currency in the write code, but judging by the comment on the mock we're probably stuck with it.